docs/docs/getting-started/installation.mdx

```diff
@@ -55,3 +46,17 @@ Checkout the latest stable version [here](https://mapperly.riok.app).
 :::
 
 </Conditional>
+
+:::info
+It is possible to remove Mapperly runtime references completely by setting `PrivateAssets="all"` and `ExcludeAssets="runtime"` on the `PackageReference`.  
+`PrivateAssets="all"` ensures that projects referencing this project do not also get a reference to `Riok.Mapperly`.  
+`ExcludeAssets="runtime"` ensures that the Mapperly .dll files are not copied to the build output.
+
+Removing the runtime dependency comes with the disadvantage that whenever the runtime tries
+to load a referenced attribute from `Riok.Mapperly.Abstractions` a `FileNotFoundException` is thrown.
+This is, for example, the case if `typeof(MyMapper).GetCustomAttributes()` is called.
+
+In Blazor wasm applications `<BlazorWebAssemblyLazyLoad Include="Riok.Mapperly.Abstractions.dll" />` can be added to lazily load `Riok.Mapperly.Abstractions`
```

Comment:
Wasm is usually abbreviated with an uppercase W

Reply:
Fixed.